node-runner-cli/templates/standalone-gateway-compose.yml.j2

```diff
@@ -0,0 +1,66 @@
+version: '3.8'
```

Comment:
Do we need another file? We were constructing the standalone configuration based on the node config

Reply:
@shambupujar It is possible to use the same file. 
My concern was that the template could become to complex when adding more if/else that would stop the core node and nginx from starting.